tests/test_convert_lit_checkpoint.py

```diff
@@ -236,13 +236,13 @@ def test_against_original_open_llama_3b():
 
 
 @torch.inference_mode()
-def test_against_hf_phi():
+def test_against_hf_phi_1_5():
     workdir = wd / "tests" / "reference_models"
     workdir.mkdir(parents=True, exist_ok=True)
     file_paths = [workdir / "original_phi_1_5.py", workdir / "configuration_phi.py"]
     urls = [
-        "https://huggingface.co/microsoft/phi-1_5/raw/24f9ea14df973a49a0d87c16d04df88d90067468/modeling_phi.py",
-        "https://huggingface.co/microsoft/phi-1_5/raw/24f9ea14df973a49a0d87c16d04df88d90067468/configuration_phi.py",
+        "https://huggingface.co/microsoft/phi-1_5/raw/main/modeling_phi.py",
```

Comment:
we could also just explicitly set the current revision here

Reply:
In `scripts/download.py` (after this PR is merged) the latest versions of Phi models will be downloaded.
So, in my opinion, tests should use the corresponding versions of modeling/config files.

Plus, if to use the current revision, we will lose all the fan of tests being falling when Microsoft decides to update modeling file one more time 😆.